google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/it/DirectPathFallbackIT.java

```diff
@@ -225,23 +230,30 @@ public void connect(
         ChannelPromise promise)
         throws Exception {
 
-      this.isIPv6 =
-          (remoteAddress instanceof InetSocketAddress)
-              && ((InetSocketAddress) remoteAddress).getAddress() instanceof Inet6Address;
+      if (remoteAddress instanceof InetSocketAddress) {
+        InetAddress inetAddress = ((InetSocketAddress) remoteAddress).getAddress();
+        String addr = inetAddress.getHostAddress();
+        if ((inetAddress instanceof Inet6Address && addr.startsWith(DP_IPV6_PREFIX))
+          || (inetAddress instanceof Inet4Address && addr.startsWith(DP_IPV4_PREFIX))) {
+          isDpAddr = true;
+        }
+      }
 
-      if (!(isIPv6 && blackholeIPv6.get())) {
+      if (!(isDpAddr && blackholeDpAddr.get())) {
         super.connect(ctx, remoteAddress, localAddress, promise);
       } else {
         // Fail the connection fast
+        System.out.println(">>>>>>>>>>>>>>>>>>>>>>>>>>> Failing connection");
         promise.setFailure(new IOException("fake error"));
       }
     }
 
     @Override
     public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception {
-      boolean dropCall = isIPv6 && blackholeIPv6.get();
+      boolean dropCall = isDpAddr && blackholeDpAddr.get();
 
       if (dropCall) {
+        System.out.println(">>>>>>>>>>>>>>>>>>>>>>> Dropping call.");
```

Comment:
Please remove the printlns.

If they are needed then maybe convert them into log.trace() or debug()

Reply:
Those were for my local debugging. Removed.